engine/src/main/java/org/operaton/bpm/engine/impl/batch/variables/BatchSetVariablesHandler.java

```diff
@@ -51,7 +51,7 @@ public void executeHandler(BatchConfiguration batchConfiguration,
 
     for (String processInstanceId : processInstanceIds) {
       commandContext.executeWithOperationLogPrevented(
-          new SetExecutionVariablesCmd(processInstanceId, variables, false, true));
+          new SetExecutionVariablesCmd(processInstanceId, variables, false, true, false));
```

Comment:
I wonder if it would be nice to introduce variables / constants here, because the trailing parameters `false`, `true`, `false` don't carry a lot of semantics. @hauptmedia @kthoms what do you think, would that make the code more readable, or are IDEs helping us here, anyways?

Reply:
My understanding of this patch (without looking too much into the details) is that this a patch within a bigger effort to unify the behaviour for all batch operations until 7.24 LTS. So from a functional point of view it is important that we follow these changes. Refactoring the internal command structure seems to me another task where we should have a broader look how the command api can be improved in general.

Reply:
Works for me, thanks!